packages/client/src/generation/TSClient/Model.ts

```diff
@@ -329,29 +342,36 @@ export class ModelDelegate implements Generatable {
   ) {}
   public toTS(): string {
     const { fields, name } = this.outputType
-    const mapping = this.dmmf.mappingsMap[name]
-    if (!mapping) {
-      return ''
-    }
-    const model = this.dmmf.modelMap[name]
 
-    const actions = Object.entries(mapping).filter(
-      ([key, value]) => key !== 'model' && key !== 'plural' && key !== 'aggregate' && key !== 'groupBy' && value,
-    )
+    const mapping = this.dmmf.mappingsMap[name] ?? { model: name, plural: `${name}s` }
+    const model = this.dmmf.typeModelMap[name]
```

Comment:
Maybe could be better if we also rename the const? It can be confusing now I think.
From `const model` to `const modelType` maybe?